File: 11399.py
# 5
# 3 1 4 3 2

import sys
input = sys.stdin.readline

a = int(input())
b = list(map(int, input().split()))
b.sort()

# 1 2 3 3 4

answer = 0
for i in range(a):
    answer += sum(b[0:i+1])

print(answer)

File: javascript/11399.js
// 5
// 3 1 4 3 2

const fs = require("fs");
const io = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs.readFileSync(io).toString().trim().split("\n");

const N = parseInt(input.shift());
let numbers = input[0].split(" ").map(Number);
numbers.sort((a, b) => a - b);

let answer = 0;

for (let i = 0; i < N; i++) {
  let tempSum = 0;
  for (let j = 0; j <= i; j++) {
    tempSum += numbers[j];
  }
  answer += tempSum;
}
console.log(answer);
